docs/lib/graphqlapi/fragments/native_common/authz/common.md

```diff
@@ -158,6 +162,26 @@ If you are using Cognito's user pool as the authorization type, this will by def
 
 <inline-fragment platform="ios" src="~/lib/graphqlapi/fragments/ios/authz/21_oidc.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/graphqlapi/fragments/android/authz/21_oidc.md"></inline-fragment>
+### Lambda Authorizer
+
+Update the `amplifyconfiguration.json` file and code snippet as follows:
```

Comment:
nit: can we maybe rephrase this to point out that devs only need to update the `authorizationType` key?